action.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Install PocketIC server'
```

Comment:
I'd name this file `pocketic-install.yml` and put it in `.github/actions`.

Reply:
The idea is that you can import this repo in other workflows like:
```
steps:
    - name: Install PocketIC server
      uses: dfinity/pocketic@main
```
Would that still work if the action.yml is under .github/actions?

Reply:
I think it would. This can and should be tested.
```
steps:
  - name: Install PocketIC server 
    uses: dfinity/pocketic/.github/actions/pocketic@main
```

Reply:
isn't `dfinity/pocketic/.github/actions/pocketic@main` unnecessarily complicated compared to `dfinity/pocketic@main`?